services/dataImport.ts

```diff
@@ -0,0 +1,23 @@
+import googleapis, {google} from 'googleapis';
+
+
+const auth = new google.auth.GoogleAuth({
```

Comment:
declare this auth object only once, in a separate file, export it, and use it in both places where it is currently declared